run_dir/static/js/pricing_quote.js

```diff
@@ -604,6 +604,10 @@ app.component('v-pricing-quote', {
                   <button type="submit" class="btn btn-primary" id="generate_quote_btn" v-on:click="generate_quote('save')" :disabled="this.invoice_downloaded"> Save Agreement</button>
                 </div>
               </div>
+              <div v-if="origin === 'Agreement'" class="ml-n1 mt-5">
```

Comment:
Put the notes inside a card that can be expanded and collapsed.

Reply:
Hm like an accordion? Are we using such a card somewhere?

Reply:
There should be one on the Projects page for customising the filters.

Reply:
You mean this one?

![Skärmavbild 2023-04-06 kl  14 27 42](https://user-images.githubusercontent.com/37932118/230378715-58baeb27-54b2-4511-a771-60f5b92c2a77.png)

Are we sure we want it to be an accordion? What if there are many notes, it will push down the agreement table and not look very pretty. Isn't a scrollbar better? Just my opinion 👼

Reply:
I'll see how it looks!

Reply:
I mimicked the logic of the "More options" accordion already present on the same page, maybe the aesthetic should be consistent with the rest of the page? I don't know what you think. 

I can put it up on stage again!